cmd/homi/setup/cmd.go

```diff
@@ -786,6 +810,66 @@ func makeSCNodes(num int, isWorkOnSingleHost bool) ([]*ValidatorInfo, []string)
 	return scn, scnNodeKeys
 }
 
+func makeSPNNodes(num int, isWorkOnSingleHost bool) ([]*ValidatorInfo, []string) {
```

Comment:
Can we merge these 3 functions? :)
makeSCN/SPN/SENNodes?

Reply:
I'd like to do refactoring at the next PR.